websites/V/Vrc.tl/presence.ts

```diff
@@ -0,0 +1,53 @@
+const presence = new Presence({
+		clientId: "1201575439974543410",
+	}),
+	browsingTimestamp = Math.floor(Date.now() / 1000),
+	regexes = {
+		detail: /\/admin\/event\/detail\/(\d+)/,
+		index: /\/([0-9]+(\|[0-9]+)+)*/,
+	};
+
+presence.on("UpdateData", async () => {
+	const presenceData: PresenceData = {
+		largeImageKey: "logo",
+		smallImageKey: "search",
+		startTimestamp: browsingTimestamp,
+		buttons: [
+			{
+				label: "View Timeline",
+				url: "https://vrc.tl/",
+			},
+		],
```

Comment:
[Buttons](https://github.com/PreMiD/Presences/blob/main/.github/CONTRIBUTING.md#presencets) that lead to main pages are not allowed.